dist/AbstractApi.js

```diff
@@ -6,30 +6,56 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const Enums_1 = require("./Enums");
```

Comment:
i just noticed this 
but why do you call this Enums_1?

Reply:
that's just how the compiler spits out the reference. Everything in `dist` is compiled, and shouldn't be looked at as written code.